scripts/release-docker.sh

```diff
@@ -22,7 +22,7 @@ function usage() {
   echo "  -n, --network             The network configuration to use (devnet or mainnet). Default=devnet"
   echo "  -b, --branch              The branch of the mina repository to use for staged docker builds. Default=compatible"
   echo "  -r, --repo                The currently used mina repository"
-  echo "      --deb-codename        The debian codename (bullseye or focal) to build the docker image from. Default=stretch"
```

Comment:
@dkijania can you remove release-docker.sh? `git grep release-docker.sh` gives me an empty output.